src/main/java/uk/gov/pay/products/persistence/dao/PaymentDao.java

```diff
@@ -57,4 +58,15 @@ public Optional<PaymentEntity> findByGovukPaymentId(String govukPaymentId) {
                 .setParameter("govukPaymentId", govukPaymentId)
                 .getResultList().stream().findFirst();
     }
+
+    public List<PaymentEntity> getPaymentsForDeletion(ZonedDateTime maxDate, int maxNumberOfPayments) {
+        String query = "SELECT payment FROM PaymentEntity payment WHERE payment.dateCreated < :maxDate " +
```

Comment:
Can the order ASC/DESC be specified too so it is clear from the statement?

Reply:
Sure thing. Done.